### PR TITLE
CtrlP Support.

### PR DESCRIPTION
**Pull request by tracyone**
#105

### CODE REVIEW

**Comment by MattesGroeger** [comment]
Thanks for your contribution @tracyone! Could you please also add the corresponding documentation to https://github.com/MattesGroeger/vim-bookmarks/blob/master/doc/bookmarks.txt? Will merge it asap then 👍

**Comment by tracyone** [comment]
@MattesGroeger finish,please check.

File: doc/bookmarks.txt [hunk 1]
```diff
@@ -318,3 +340,4 @@ c. Why aren't any signs showing at all?
 >
   :echo has('signs')
 <
+vim:ft=help:et:ts=2:sw=2:sts=2:norl
```
Comment (MattesGroeger) [inline doc/bookmarks.txt]: Was this added on purpose?

Comment (tracyone) [inline doc/bookmarks.txt]: yes.this is vim modeline,will override settings in vimrc

Comment (MattesGroeger) [inline doc/bookmarks.txt]: Ah, didn't know that 👍

**Pull request merged**